x/txfees/keeper/feetokens.go

```diff
@@ -23,15 +23,34 @@ func (k Keeper) ConvertToBaseToken(ctx sdk.Context, inputFee sdk.Coin) (sdk.Coin
 		return sdk.Coin{}, err
 	}
 
-	spotPrice, err := k.spotPriceCalculator.CalculateSpotPrice(ctx, feeToken.PoolID, feeToken.Denom, baseDenom)
+	spotPrice, err := k.spotPriceCalculator.CalculateSpotPrice(ctx, feeToken.PoolID, baseDenom, feeToken.Denom)
```

Comment:
Were the arguments here just in the wrong order before / we not have tests checking this?

Reply:
Ah your right, the argumernts were certainly reversed! (The test had the wrong expectation coded)